rustfmt-stable.toml

```diff
@@ -13,7 +13,7 @@
 # attr_fn_like_width = 70            # Leave implicit
 # chain_width = 60                   # Leave implicit
 edition = "2018"
-fn_args_layout = "Tall"
+fn_params_layout = "Tall"
```

Comment:
These changes were made because the original name was deprecated in the latest nightly rustfmt, and I needed to make them for `just format` to work properly.